syntax = "proto3";

import 'files/common/common.proto';

message RecipeDisplayGroups {
  repeated RecipeDisplayGroup recipe_display_groups = 1;
  message RecipeDisplayGroup {
    ObjectIdentity identity = 1;
    RecipeDisplayGroupInfo info = 2;
  }

  message RecipeDisplayGroupInfo {
    string display_group_type = 1;
    string name_placeholder = 2;
    int32 sort_priority = 3;
  }
}
